src/components/HeaderIdig.vue

```diff
@@ -150,4 +145,9 @@ export default {
   background-color: #28a745;
   border-color: #28a745;
 }
+.input-header {
+  background: #212529;
+  border: 0px;
+  width: 6em;
+}
```

Comment:
@AlZorba We both forgot to change the `<style>` to `<style scoped>`
If you do some other change on this PR, you can do this too. Otherwise another time is fine